graphql-protocol/src/main/java/com/linecorp/armeria/server/graphql/protocol/AbstractGraphqlService.java

```diff
@@ -214,4 +275,13 @@ private static HttpResponse unsupportedMediaType() {
                                "Unsupported media type. Only JSON compatible types and " +
                                "application/graphql are supported.");
     }
+
+    private static HttpResponse ofFailureThrownIllegalArgumentException(IllegalArgumentException ex) {
+        final String message = ex.getMessage() == null ? HttpStatus.BAD_REQUEST.reasonPhrase()
+                                                       : ex.getMessage();
+        final HttpResponse response = HttpResponse.of(HttpStatus.BAD_REQUEST,
```

Comment:
Can't we just return this `response` instead of creating `HttpResponseException`? 🤔

Reply:
I tried to change this, but I can't capture the log for the exception. Do you have any other way? 🤔

Reply:
Sorry for the late review. 😅 

>  I can't capture the log for the exception.

Do we really need to capture this `IllegalArgumentException`? I thought it would be enough returning the response with the well-described message.
Also, we shouldn't use `HttpResponseException` internally because users cannot manipulate the `HttpResponse` using a decorator. 
https://github.com/line/armeria/pull/4117

Reply:
Oh, I didn't know. 🤣 

If so, let's change this to using the response body to validate the exception.